packages/fx-core/src/plugins/resource/bot/v2/index.ts

```diff
@@ -51,7 +51,7 @@ export class BotPluginV2 implements ResourcePlugin {
   displayName = "Bot";
   @Inject(ResourcePlugins.BotPlugin)
   plugin!: TeamsBot;
-  impl = new TeamsBotV2Impl();
+  impl = new TeamsBotV2Impl(this.name);
```

Comment:
Since the value of name is the plugin id, do we need to pass it as param or get it from const?